### PR TITLE
Essay Submission: A/B testing

### PR DESCRIPTION
**Pull request by johennin**
## Essay: A/B Testing Social Networks and Online Marketplaces: The Problem of Network Effects

Proposal PR [#979](https://github.com/KTH/devops-course/pull/979)

### Members
Name: Carl Piehl

Mail: cpiehl@kth.se

Github: [cpiehl1](https://github.com/cpiehl1)

Name: Johan Henning

Mail: johennin@kth.se

Github: [johennin](https://github.com/johenninn)

### Content

This essay dives deeper into some of the problems that A/B testing can touch on such as network effects on various social media sites and online marketplaces.

### CODE REVIEW

**Comment by iriediese** [comment]
## Feedback Members:
Hugo Heyman, hheyman@kth.se  
Github: [hallon-heyman](https://github.com/hallon-heyman)

Theodor Moise, moise@kth.se  
Github: [iriediese](https://github.com/iriediese)

# Feedback
## Overall
The essay provides good insight about A/B testing, giving examples of potential pitfalls, as well as how they are generally handled by different software companies. The word count fits the limit. Structurally, the essay is well-split among different sections, all of which bring relevant contributions to the main topic. There are 10 references, all pointing to good, relevant material.

## Title
Title is accurate, specific, and effective in describing the scope of the essay. It is a little long, but that might be necessary considering the topic at hand.

## Introduction
The introduction provides a brief description of A/B testing as a practice. Its purpose is then elaborated with further examples, showing how it changed over time. Finally, the introduction ends with an overview of what is going to be discussed in more detail, which is a welcome addition. The section is neither too long or too short, and seems to include everything it needs to. Great work here.

## What is A/B Testing?
This section explains how A/B testing is done in practice. It starts by introducing the necessary terminology, and goes on to show an example. Formatting is consistent, figures are well-placed and relevant. 

## Network Effects
Description of the SUTVA and network effects. Well-written, overall. Maybe you could tie it to the previous section a bit more (the change of topic feels a bit sudden). Consider briefly mentioning it before, and then properly defining it in its own section (here) afterwards.

## Social Networks
This section shows examples of network effects, and can be improved a little. Consider replacing *in the first* with *in the first place*, or just *first*. Then, you start out by mentioning to examples of how network effects might appear *in social networks*. Consequently, both examples should be of social networks. However, the GCP is not really a social network. Maybe rephrase the intro sentence. Finally, you specify that the second example is about the GCP, but you do not mention the social network the first example is about. Then, in the following subsection, you mention LinkedIn and OkCupid. You could maybe also name that first one, for consistency. In case it's Facebook, I found that Facebook also does A/B testing for ad campains. You can find more about this [here](https://www.facebook.com/business/help/166313650471318?recommended_by=1738164643098669). Content-wise, the section is good, though.

## Online Marketplaces
This section shows how network effects are tackled by online marketplaces. The first subsection here is clear and relevant. The Ebay example is well-put. Ridesharing, however, is quite different from an online marketplace. Firstly, it is mostly based on trading a service, and secondly, the product is the same every time, just like you said. In this case, it probably deserves its own section, alongside social networks and online marketplaces. Anyway, the method used by Uber is also quite interesting. Good addition.

## Conclusions
Typo on the second line, *group* should be plural. Also, remove the comma after *looked at*. Great conclusion, either way. It sums everything up neatly. Some previously-mentioned examples are referred to one last time. The section is just the right size. Good work.

**Comment by SophieHYe** [comment]
Thanks for the final essay submission. I am now merging your PR and we will grade your work based on the final version.

**Comment by khesoem** [comment]
Feedback comments applied in: #1418
